### PR TITLE
Support a Native Jenkins Installation

### PR DESCRIPTION
**Pull request by rodibrin**
According to the [https://github.com/SAP/jenkins-library/issues/781](user permission issue), the image should be adjusted so that it can run under any user id.

For testing purposes, the adjusted image can be pulled from rstengel/cf-cli:latest

Test instances:

cx-server:          https://d046169piperj383.jaas-gcp.cloud.sap.corp/job/ppiper-cx-default/
native-jenkins:  http://10.47.228.211:8080/job/ppiper-cx-default/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,3 +28,7 @@ RUN cf add-plugin-repo CF-Community https://plugins.cloudfoundry.org && \
     cf install-plugin ${MTA_PLUGIN_URL} -f && \
     cf install-plugin Create-Service-Push -f -r CF-Community && \
     cf plugins
+
+# allow anybody to read/write/exec at HOME
+RUN chmod -R o+rwx "${USER_HOME}"
+ENV HOME=${USER_HOME}
```
Comment (rkamath3) [inline Dockerfile]: Please add newline

**Comment by fwilhe** [comment]
We cannot merge without a green travis build. Please create a PR from the upstream repo.

**Comment by marcusholl** [comment]
> We cannot merge without a green travis build. Please create a PR from the upstream repo.

It is sufficient to push the HEAD of the PR branch to `refs/heads/it`. There is no need for opening PRs in the base repo. Non-team members are even not able to do so. I will run the integration tests for the PR here.

**Comment by fwilhe** [comment]
Thanks, I always have a hard time to remember if we set this up in all repos.

**Comment by marcusholl** [comment]
Sorry, wrong repo. This repo is not prepared for that.

So we can only work with PRs in the base repo, which is IMO really crazy. Should we provide everybody with the required privileges in the base repo?

**Comment by fwilhe** [comment]
>So we can only work with PRs in the base repo, which is IMO really crazy. Should we provide everybody with the required privileges in the base repo?

I'm not happy with that situation, but I also want _real_ tests in those repos, thus requiring credentials. I'm not sure if a better solutions for this exists.

**Comment by rodibrin** [comment]
re-created the pull request from master repository due to Travis restrictions.

**Pull request closed without merging**